apps/explorer/src/components/Layout/index.tsx

```diff
@@ -95,18 +95,29 @@ export function Layout() {
                             }}
                             toastOptions={{
                                 duration: 4000,
-                                success: {
-                                    icon: null,
-                                    className:
-                                        '!bg-success-light !text-success-dark',
-                                },
-                                error: {
-                                    icon: null,
-                                    className:
-                                        '!bg-issue-light !text-issue-dark',
-                                },
                             }}
-                        />
+                        >
+                            {(toast) => {
+                                let variant: BannerProps['variant'] = undefined;
+                                switch (toast.type) {
```

Comment:
Usually defining an ObjectMap `Record<ToastType, BannerType>` or Partial ToastTypes is cleaner and easier to extend. Like
```
obj = {success: 'positive', ...}

const variant = obj[toast.type] || 'neutralWhite';
```